### PR TITLE
0.9.0

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

File: languages/python/django-oso/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-oso~=0.8
+oso~=0.9.0
```
Comment (gj) [inline languages/python/django-oso/requirements.txt:1]: Calling out this change

File: languages/python/sqlalchemy-oso/Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .PHONY: build test test-requirements fmt lint
 build:
 	rm -rf build
-	python setup.py develop
+	pip install -e .[flask]
```
Comment (gj) [inline languages/python/sqlalchemy-oso/Makefile:4]: @plotnick to test whether these brackets need to be escaped (or the entire `.[flask]` quoted)

Comment (plotnick) [reply languages/python/sqlalchemy-oso/Makefile:4]: Seems ok.

**Review by plotnick — APPROVED** [review]
Release! Release!

**Pull request merged**